src/components/Recent.tsx

```diff
@@ -96,6 +96,7 @@ export default function Recent<T>(props: Props<T>): React.ReactElement {
     <RecentCard label={label} resource={resource}>
       <ResourceContext.Provider value={resource}>
         <List
+          storeKey={`recent-${resource}` }
```

Comment:
@salony75way - can you clarify what the pagination problem was please?

Reply:
List params corresponding to resources are stored to local storage so when we open bathes list after welcome page then the recent batches pagination was applied to batch list and also batches are sorted by descending order.
This was vice versa so just changed the storeKey for recent batches.